### ISSUE TITLE
[Request] Update of redbean's `sqlite3` version in `lsqlite3` module

### ISSUE THREAD

**Issue by chrisokuda** [jart/cosmopolitan#508]
SQLite `3.35.5` is very old, almost 18 months. Requesting it be updated to the most recent version possible (`3.39.2` at submission time).

From `redbean-2.0.14.com -i`:

```lua
:> require("lsqlite3").version()
"3.35.5"
```

**Comment by pkulchenko** [comment]
I started looking into this, but it may be worthwhile to wait until [wal2 changes](https://www.sqlite.org/cgi/src/timeline?r=wal2) are merged into master to avoid re-doing these changes. Although I'm not sure when to expect that.

**Comment by jxonas** [comment]
The wal2 mode has almost 5 years now. The branch is kept up to date with trunk periodically but there is no indication of it being merged into master any time soon (at least I couldn’t find in the docs or the oficial forum).

**Comment by pkulchenko** [comment]
@chrisokuda, @jxonas, I got sqlite upgraded to 3.40 in redbean (https://github.com/pkulchenko/cosmopolitan/tree/sqlite-340), but there is still some work to be done to migrate some of the changes that have been applied in cosmopolitan to the original version. I plan to apply them over the next couple of days.

I've tested it on a couple of fairly complex redbean-based application I have and everything working so far, but if you get a chance to run it, let me know if everything looks good for you.
